src/test/resources/maps/50x50_MenuLevel.crf

```diff
@@ -0,0 +1,17 @@
+# Test level with an entity and light
```

Comment:
That was the idea, but after your comment I checked if it was really needed and it turned out I can remove the copy safely.